### PR TITLE
ENH: Allow reading file-like objects for EpochsFIF

### PR DESCRIPTION
**Pull request by paulroujansky**
## Description
In the light of https://github.com/mne-tools/mne-python/pull/6291 which enabled to read `mne.io.fiff.raw.Raw` FIF object from file-like object, the hereby PR aims at extending this feature to `mne.epochs.EpochsFIF` object (cf. issue https://github.com/mne-tools/mne-python/issues/6009).

### CODE REVIEW

**Comment by agramfort** [comment]
nice !

can you add an entry in what's new doc page?

@larsoner I'll let you merge this one.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -214,6 +214,8 @@ Changelog
 
 - Add ``mne.realtime.MockLSLStream`` to simulate an LSL stream for testing and examples by `Teon Brooks`_
 
+- Add support for file-like objects in :func:`mne.epochs.read_epochs` as long as preloading is used by `Paul Roujansky`_
```
Comment (larsoner) [inline doc/whats_new.rst]: Can you add a link at the bottom? It's probably why CricleCI complains

Comment (paulroujansky) [reply doc/whats_new.rst]: My bad, thanks for that @agramfort

**Comment by agramfort** [comment]
thx @paulroujansky

**Pull request merged**